rebar.config

```diff
@@ -41,7 +41,10 @@
             {test, [{relx, [{dev_mode, true},
                             {include_erts, false},
                             {include_src, true}]},
-                    {deps, [{meck, ".*", {git, "git://github.com/eproxus/meck.git", {tag, "0.8.6"}}}
+                    {dist_node, [{setcookie, 'epoch_cookie'},
+                                 {sname, epoch_ct}]},
+                    {deps, [{meck, ".*", {git, "git://github.com/eproxus/meck.git", {tag, "0.8.6"}}},
+                            {erlexec, ".*", {git, "git://github.com/saleyn/erlexec.git", "HEAD"}}
```

Comment:
In order to prevent unexpected breakages, may I suggest `{ref, "97a5188"}`.

Reply:
Agreed.

Reply:
I've pushed a change for this